ESX-EVRAMigrationJobScript.ps1

```diff
@@ -0,0 +1,164 @@
+            $DtHostUserName = "root"
+        }
+    
+        $DtHostPassword = Read-Host -AsSecureString -Prompt "Please enter the ESX host user password"
+
+        $HostCredential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $DtHostUserName, $DtHostPassword
+
+        # Type of workload you will be protecting and type of job you will be creating
+        $DtWorkloadType = Read-Host -Prompt 'Please enter the type of job you will be creating [default = VraMove]'
+        if (!$DtWorkloadType) {
+            $DtWorkloadType = "VraMove"
+        }
+        $DtJobType = $DtWorkloadType
+
+        # Create source and target objects
+        $DtSource = New-DtServer -Name $DtSourceName -UserName $DtSourceUserName -Password $SourceCredential.GetNetworkCredential().Password
+        $DtTarget = New-DtServer -Name $DtTargetName -UserName $DtTargetUserName -Password $TargetCredential.GetNetworkCredential().Password
+
+        # Create ESX host appliance object
+        # If you are using vCenter, specify your vCenter.
+        # Only specify an ESX host if you are using ESX standalone.        
+        $VimTarget = New-DtServer -Name $DtHostName -Username $DtHostUserName -Password $HostCredential.GetNetworkCredential().Password -Role TargetVimServer
+
+        $OtherServers = @($VimTarget)
+
+        # Create a workload
+        $DtWorkloadGUID = New-DtWorkload -ServiceHost $DtSource -WorkloadTypeName $DtWorkloadType
+
+        # This workload, by default, selects all volumes for protection
+        # If desired, exclude any volumes from protection, however, be careful
+        # when excluding data as it may compromise the integrity of your installed applications
+        # Uncomment and use the following line, substituting G:\for the volume you want to exclude
+        # Repeat the line to exclude multiple volumes
+        # Set-DtLogicalItemSelection -ServiceHost $DtSource -WorkloadId $DtWorkloadGuid -LogicalPath "G:\" -Unselect
+
+        # Get the workload definition including the workload and logical items
+        $DtWorkload = Get-DtWorkload -ServiceHost $DtSource -WorkloadId $DtWorkloadGUID
+
+        # Get the default job options that will be used to create the job
+        $DtJobOptions = Get-DtRecommendedJobOptions -ServiceHost $DtTarget -Source $DtSource -JobType $DtJobType -Workload $DtWorkload -OtherServers $OtherServers
+
+        # Create the job
+        $timestamp = Get-Date -Format "yyyyMMdd.HHmmss"
+        # Generate a unique replica name
+        Write-Host "Usable tags for replica name:
+        {YYYY} = 4 digit year
+        {MM} = 2 digit month
+        {DD} = 2 digit day
+        {HH} = 2 digit hour
+        {MN} = 2 digit minutes
+        {SS} = 2 digit seconds
+        {MS} = milliseconds
+        {IP} = Current IP of the VM to be migrated
+        If no name is entered, a default name will be generated in the following format d42-carbonite-[source ip].[YYYYMMDD.HHMMSS]"
```

Comment:
Default needs to be:
carbonite-[source ip].[YYYYMMDD.HHMNSS]

Just change to the MN for minutes.

Reply:
Done.